src/actions/Intel/generalActions.js

```diff
@@ -0,0 +1,175 @@
+//#region > Registration
+/**
+ * Append Source Objects
+ *
+ * @param sourceList A source object
+ * @description Hands source list over to intel
+ */
+const appendSourceObjects = (sourceList) => {
+  return async (intel) => {
+    try {
+      return intel.appendList(sourceList);
+    } catch (ex) {
+      return {
+        errorCode: 2,
+        message: "Appending source objects failed",
+        raw: ex,
```

Comment:
rawr.

<img src="https://user-images.githubusercontent.com/21159914/84789386-76e5b980-aff0-11ea-92a0-9833b49a102d.png" alt="Furrytales" height="250"/>

<sub><sup>Disclaimer: I do not approve the existance of furries</sup></sub>

Reply:
![image](https://user-images.githubusercontent.com/52858351/85020988-7163ad00-b171-11ea-9d78-272562879edf.png)

Reply:
Does anything has to be changes at this code part??

Reply:
nep

Reply:
Best thread on this platform. ☠